src/main/java/com/ibm/g11n/pipeline/client/ServiceClient.java

```diff
@@ -626,4 +629,137 @@ public abstract TranslationConfigData getTranslationConfig(String sourceLanguage
      */
     public abstract void deleteTranslationConfig(String sourceLanguage, String targetLanguage)
            throws ServiceException;
+
+
+    //
+    // {serviceInstanceId}/v2/trs APIs
+    //
+
+    /**
+     * Returns a map containing <code>TranslationRequestData</code> indexed by translation
+     * request IDs.
+     * 
+     * @return A map containing <code>TranslationRequestData</code> indexed by translation.
+     * @throws ServiceException when the operation failed.
+     */
+    public abstract Map<String, TranslationRequestData> getTranslationRequests() throws ServiceException;
+
+    /**
+     * Returns the translation request specified by the translation request ID.
+     * 
+     * @param trId  The translation request ID.
+     * @return  The translation request data.
+     * @throws ServiceException when the operation failed.
+     */
+    public abstract TranslationRequestData getTranslationRequest(String trId)
+            throws ServiceException;
+
+    /**
+     * Creates a new translation request.
+     * 
+     * @param newTranslationRequestData The new translation request.
+     * @return  The translation request created by this operation.
+     * @throws ServiceException when the operation failed.
+     */
+    public abstract TranslationRequestData createTranslationRequest(NewTranslationRequestData newTranslationRequestData)
+            throws ServiceException;
+
+    /**
+     * Updates the translation request.
```

Comment:
If the TR is in SUBMITTED state already, updateTranslationRequest() will fail. The REST server rejects the request for updating the TR.

A new TR can be created with either DRAFT or born SUBMITTED state. A user can change TR in DRAFT state to SUBMITTED state if the TR has valid values in all necessary fields. Then, this is the point when translation package should be created and sent to translators.